modules/lss-showChatbuttonAbove/showChatbuttonAbove.user.js

```diff
@@ -0,0 +1 @@
+$('#navbar-main-collapse > ul').append('<li><a class="lightbox-open" href="' + $("#navbar_profile_link").attr("href") + '">' + user_id + '</a></li>');
```

Comment:
Hier gibt es eine Abhängigkeit zur Creditserweiterung. Dieses Script würde nur gemeinsam mit dem anderen funktionieren. Und auch nur dann, wenn der Browser es in der richtigen Reihenfolge ausführt.

Bitte sorge dafür, dass jedes Modul isoliert für sich funktionsfähig ist (Kapselung).

Reply:
Danke, wieso ist mir sowas nicht aufgefallen?

Reply:
Diesen Einwand ziehe ich zurück. Ich habe gesehen, dass user_id eine global verfügbare Variable aus dem Basisspiel ist. Gibt also doch keine Abhängigkeiten, wenn ich das richtig sehe.